Readme.md

```diff
@@ -10,8 +10,20 @@ a cookie for tracking sessions.
 + [fastify-caching](https://www.npmjs.com/package/fastify-caching): used to
 store the session data via the `fastify.cache` decorator.
 
+
+**Installation:**
+
+```
+npm install fastify-server-session fastify-cookie fastify-caching
+```
+
 ## Example
 
+### Server Side Storage
+
+Using this implementation the sessions will be stored in the Fastify instance
+making the server statefull.
```

Comment:
```suggestion
Using this implementation the sessions will be stored in memory on the Fastify instance
making the server stateful. This is not recommended for production. It will not share
state among multiple instances of the same application.
```